analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConditionalsWithSameCondition.CSharp9.cs

```diff
@@ -60,6 +60,11 @@ public void Foo()
             if (ch is >= 'a' and <= 'z') { } // Noncompliant
             if (ch is <= 'z' and >= 'a') { } // Compliant, even when the semantics is the same
 
+            if (ch is (>= 'a' and <= 'z') or (>= 'A' and <= 'Z') or '.' or '?') { }
+            if (ch is (>= 'a' and <= 'z') or (>= 'A' and <= 'Z') or '.' or '?') { } // Noncompliant
+            if (ch is (>= 'a' and <= 'z') or (>= 'A' and <= 'Z') or '?' or '.') { } // Compliant, even when the semantics is the same
```

Comment:
I really suggest to use `FN` instead of `Compliant` for these "even when" cases

Compare this with lines 76-78 - it _really is compliant_ - here it's just an FN. If we would raise, it would help developers